addons/l10n_au/data/template/account.tax-au.csv

```diff
@@ -1,114 +1,114 @@
+"au_tax_purchase_input_service_tpar_no_abn","10% INP TPAR NO ABN","6","Purchases for Input Taxed Sales","Purchases for Input Taxed Sales","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_input_service_tpar,au_tax_witheld","","","","","False",""
+"au_tax_purchase_private_service","10% PRIV","7","Purchases for Private use or not deductible","10% Purchases for Private Use","purchase","","percent","10.0","False","tax_group_gst_10","","base","invoice","","+G11||+G15","True",""
+"","","","","","","","","","","","","tax","invoice","","+G11||+G15","",""
+"","","","","","","","","","","","","base","refund","","-G11||-G15","",""
+"","","","","","","","","","","","","tax","refund","","-G11||-G15","",""
+"au_tax_purchase_private_service_tpar","10% PRIV TPAR","7","Purchases for Private use or not deductible","Purchases for Private use or not deductible","purchase","service","percent","10.0","False","tax_group_gst_10","","base","invoice","","+G11||+G15","False",""
+"","","","","","","","","","","","","tax","invoice","","+G11||+G15||Service","",""
+"","","","","","","","","","","","","base","refund","","-G11||-G15","",""
+"","","","","","","","","","","","","tax","refund","","-G11||-G15||Service","",""
+"au_tax_purchase_private_service_tpar_no_abn","10% PRIV TPAR NO ABN","7","Purchases for Private use or not deductible","Purchases for Private use or not deductible","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_private_service_tpar,au_tax_witheld","","","","","False",""
+"au_tax_purchase_gst_only_service","100% ONLY","8","GST Only on Imports","GST Only on Imports","purchase","","division","100.0","True","tax_group_gst_100000000","","base","invoice","","","True",""
+"","","","","","","","","","","","","tax","invoice","au_21330","+ONLY||+GST from General Ledger","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","au_21330","-ONLY||-GST from General Ledger","",""
+"au_tax_purchase_gst_only_service_tpar","100% ONLY TPAR","8","GST Only on Imports","GST Only on Imports","purchase","service","division","100.0","True","tax_group_gst_100000000","","base","invoice","","","False",""
+"","","","","","","","","","","","","tax","invoice","au_21330","+ONLY||+GST from General Ledger||Service","",""
+"","","","","","","","","","","","","base","refund","","","",""
+"","","","","","","","","","","","","tax","refund","au_21330","-ONLY||-GST from General Ledger||Service","",""
+"au_tax_purchase_gst_only_service_tpar_no_abn","100% ONLY TPAR NO ABN","8","GST Only on Imports","GST Only on Imports","purchase","service","group","100.0","","tax_group_gst_10","au_tax_purchase_gst_only_service_tpar,au_tax_witheld","","","","","False",""
+"au_tax_purchase_adj_service","100% ADJ","9","Tax Adjustments (Purchases)","Tax Adjustments (Purchases)","purchase","","division","100.0","False","tax_group_gst_100000000","","base","invoice","","+G18","True",""
```

Comment:
Label on invoice shouldn't have the parenthesis according to the spec.

Reply:
seems unintentional to me and breaking consistency with other taxes descriptions and invoice labels
![image](https://github.com/odoo/odoo/assets/160074741/834e937e-f9d9-487c-aeec-feae41f63725)

Reply:
Well you removed the parentheses from the sale one :smile:
There is no need for consistency between labels and descriptions. (but you are right about consistency between labels)
In doubt I would follow the spec.